x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/job/config/DataDescriptionTests.java

```diff
@@ -55,7 +55,6 @@ public void testVerify_GivenValidFormat() {
         description.setTimeFormat("yyyy-MM-dd HH");
     }
 
-    @AwaitsFix(bugUrl = "https://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8206980")
     public void testVerify_GivenValidFormat_Java11Bug() {
```

Comment:
I think it would be worth renaming this method, otherwise it will cause confusion in the future about whether it's expected to be a bug or not (since the only reference to the bug is in the line you've deleted).

Alternatively, the format this method tests could be moved back into the method above (which is where it was originally).